orchprovider/k8s/v1/util.go

```diff
@@ -21,6 +23,79 @@ import (
 	"k8s.io/client-go/rest"
 )
 
+type MonitoringSideCar struct {
+	// TargetIP is the IP Address of the
+	// service using which this sidecar will
+	// pull metrics info
+	TargetIP string
+
+	// Image of this sidecar
+	Image string `json:"image,omitempty"`
+
+	// SideCar is the K8s type that represents a sidecar
+	SideCar k8sApiV1.Container
+}
+
+// monSideCarTpl is the K8s type (used as a template) that gets
+// built as a sidecar
+var monSideCarTpl = k8sApiV1.Container{
+	Name:  "maya-volume-exporter",
+	Image: "openebs/m-exporter:ci",
+	Command: []string{
+		"maya-volume-exporter",
+	},
+	Args: []string{
+		"-c=http://__TARGET_IP__:9501",
```

Comment:
What address are you passing in __TARGET_IP_ and how ? Is it Jiva Controller POD's IP ?

Reply:
yes